src/test/java/com/faforever/client/update/VersionTest.java

```diff
@@ -0,0 +1,60 @@
+package com.faforever.client.update;
+
+import org.junit.Test;
+
+import java.lang.reflect.Field;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class VersionTest {
+
+  @Test(expected = NullPointerException.class)
+  public void shouldFailOnNullFromVersion() {
+    Version.shouldUpdate(null, "v1.0.0");
+  }
+
+  @Test(expected = NullPointerException.class)
+  public void shouldFailOnNullToVersion() {
+    Version.shouldUpdate("v1.0.0", null);
+  }
+
+  @Test
+  public void shouldUpdateIfRemoteIsNewer() {
+    assertTrue(Version.shouldUpdate("1.0.9", "v1.1.0"));
+    assertTrue(Version.shouldUpdate("v1.0.9", "1.1.0"));
+    assertTrue(Version.shouldUpdate("1.1.0", "v1.1.1"));
+    assertTrue(Version.shouldUpdate("1.9.9", "v2.0.0"));
```

Comment:
add hre checks for 
`1.0.9-rc1`, `...-alpha`

Reply:
Done